setup/gcp/installer/forseti_server_installer.py

```diff
@@ -47,25 +49,58 @@ def __init__(self, **kwargs):
         """
         super(ForsetiServerInstaller, self).__init__()
         self.config = ServerConfig(**kwargs)
+        self.v1_config = None
 
     def preflight_checks(self):
         """Pre-flight checks for server instance"""
 
         super(ForsetiServerInstaller, self).preflight_checks()
-
+        gcloud.enable_apis(self.config.dry_run)
+        forseti_v1_name = None
+        if not self.config.dry_run:
+            _, zone, forseti_v1_name = gcloud.get_vm_instance_info(
+                r'^forseti-security-\d+-vm$', try_match=True)
+        if forseti_v1_name:
+            utils.print_banner('Import configuration and rules from v1')
+            # v1 instance exists, ask if the user wants to port
+            # the conf/rules settings from v1.
+            self.prompt_v1_configs_migration()
+        if self.migrate_from_v1:
+            self.v1_config = v1_upgrader.ForsetiV1Configuration(
+                self.project_id, forseti_v1_name, zone)
+            self.v1_config.fetch_information_from_gcs()
+            self.populate_config_info_from_v1()
         self.determine_access_target()
         self.should_enable_write_access()
         self.format_gsuite_service_acct_id()
         self.should_grant_access()
 
-        gcloud.enable_apis(self.config.dry_run)
         self.gsuite_service_account = gcloud.create_or_reuse_service_acct(
             'gsuite_service_account',
             self.gsuite_service_account,
             self.config.advanced_mode,
             self.config.dry_run)
         self.get_email_settings()
 
+    def prompt_v1_configs_migration(self):
+        """Ask the user if they want to migrate conf/rule files
```

Comment:
nit: This should fit on one line.

Reply:
It almost fits in one line, 82 characters long in total.